manifests/0000_08_cluster-dns-operator_03-daemonset.yaml

```diff
@@ -0,0 +1,56 @@
+# Node resolver for internal registry
```

Comment:
If we deploy this as a manifest via CVO, what happens when we need to roll out a bug fix or other change?